test/libsolidity/syntaxTests/fallback/return_value_unsupported.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    fallback() external returns (bytes memory) {}
+}
+// ----
+// TypeError: (45-59): Return values for fallback functions are not yet implemented.
```

Comment:
@ekpyron @chriseth is this supposed to go in this PR?

Reply:
It's not breaking to add it later, so my plan was to postpone - even if we want it for 0.6.0 I'd make it a separate PR, this one is already large...

Reply:
Ok!